examples/systemcalls/systemcalls.c

```diff
@@ -9,15 +16,12 @@
 */
 bool do_system(const char *cmd)
 {
-
-/*
- * TODO  add your code here
- *  Call the system() function with the command set in the cmd
- *   and return a boolean true if the system() call completed with success
- *   or false() if it returned a failure
-*/
-
-    return true;
+    int sys_return = system(cmd);
+    
+    if(sys_return == -1 || sys_return == 127)
```

Comment:
@kevintom98 I think this logic is backwards, doesn't pass when I run against a revised test case at https://github.com/cu-ecen-aeld/assignment-autotest/pull/22